HOMEWORK_16/EX1.CPP

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+
+class rectangle 
```

Comment:
class declarations go into headers, implementations go into cpp

Reply:
I changed and split the code as much as possible, I hope you like this version more:)